### PR TITLE
OCLint improvements

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fix oclint step in Travis when multiple files changed.

Improves logging message on log and GitHub integration.

cc @sendhil

### CODE REVIEW

File: Scripts/run-oclint.sh [hunk 1]
```diff
@@ -9,51 +9,53 @@ xcodebuild_log_path=${temp_dir}/xcodebuild.log
...
            DSTROOT=$build_dir OBJROOT=$build_dir SYMROOT=$build_dir \
            -reporter json-compilation-database:$compile_commands_path
-           #| tee $xcodebuild_log_path
+           
 
 if [ $TRAVIS ]; then
-    echo "[*] only files changed on push";    
-    include_files=`git diff $TRAVIS_COMMIT_RANGE --name-only | grep '\.m' | tr '\n' ' -i '`
+    echo "[OCLint] only files changed on push";    
+    include_files=`git diff $TRAVIS_COMMIT_RANGE --name-only | grep '\.m' | tr '\n' '|' | sed 's/|*$/"/g'`
```
Comment (sendhil) [inline Scripts/run-oclint.sh]: Maybe I'm reading this wrong, but won't this only have an end `"` and not a starting one? e.g. I did `git diff ee020a2 --name-only  | tr '\n'g'|' |  sed 's/|*$/"/g` locally and got `Podfile|Podfile.lock"`

File: Scripts/run-oclint.sh [hunk 2]
```diff
@@ -61,18 +63,22 @@ if [ $TRAVIS ]; then
...
-      include_files="-i "$include_files      
+      include_files=' -i "'$include_files      
     else
       exclude_files="-e *"
     fi
-    echo "[*] $include_files"
-elif [ $1 ]; then
+    echo "[OCLint] analyzing these files: $include_files"
+elif [ $1 == "DIFF" ]; then
+    include_files=`git diff HEAD^ --name-only | grep '\.m' | tr '\n' '|' | sed 's/|*$/"/g'`
```
Comment (sendhil) [inline Scripts/run-oclint.sh]: Same comment as above

Comment (SergioEstevao) [inline Scripts/run-oclint.sh]: Check the following lines where I add a -I ".

File: Scripts/run-oclint.sh [hunk 3]
```diff
@@ -61,18 +63,22 @@ if [ $TRAVIS ]; then
     full_sha=`git rev-parse $sha`
     echo $full_sha
     if [ ! -z "$include_files" ]; then
-      include_files="-i "$include_files      
+      include_files=' -i "'$include_files
```
Comment (SergioEstevao) [inline Scripts/run-oclint.sh]: Here I add the open "

**Comment by sendhil** [comment]
Looks good.

**Pull request merged**